src/test/java/seedu/address/model/UniqueTagListTest.java

```diff
@@ -16,4 +23,72 @@ public void asObservableList_modifyList_throwsUnsupportedOperationException() {
         thrown.expect(UnsupportedOperationException.class);
         uniqueTagList.asObservableList().remove(0);
     }
+
+    @Test
+    public void equals_sameList_true() throws Exception {
```

Comment:
Good! This is how it should be, try to over as many methods and cases as possible.